test-cases/gutenberg/writing-flow/multiline-components.md

```diff
@@ -21,12 +21,4 @@ Test the next steps on:
 
 - Start typing and press enter to create a new line.
 - The new line should be created, without splitting the block.
-- On citations, pressing Enter at the end of a citation splits the block, but pressing enter in the middle of a citation creates a new line.
-- Check on HTML mode that the resulting HTML code is correct:
-  - _QuoteV1_: `<p>` tags per paragraph. **(This is for QuoteV1 _only_. QuoteV2 uses inner blocks instead)**
-  - _Quote citation_: `<br>` on line-breaks.
-  - _Verse_: `<br>` on line-breaks.
-  - _Preformatted_: `<br>` on line-breaks.
-  - _Code_: (Invisible `\n`) new line character.
-  - _Pullquote_: `<br>` on line-breaks.
-  - _Pullquote citation_: `<br>` on line-breaks.
+- On citations, pressing Enter at the end of a citation splits the block, but pressing enter in the middle of a citation creates a new line.
```

Comment:
> Aside from the above assertion, I believe the tests [cover everything else](https://github.com/WordPress/gutenberg/blob/7b68d0a3540abadf0593b27de404f7489b68d119/packages/block-library/src/quote/test/edit.native.js#L76-L86). Is that your rationale to retain the Mutli-line Components test case?

I was tempted to remove the entire test case. But I noticed in the internal Gutenberg Mobile Test Cases document that we marked it as "Need tests", specifically E2E tests. There's also the following note:

> The core functionality (expected markup) could be tested with integration tests. We should likely cover some of the known issues with e2e tests to cover nuanced visual regressions specific to each platform. 

That's my rationale to retain it.

Reply:
> I was tempted to remove the entire test case. But I noticed in the internal Gutenberg Mobile Test Cases document that we marked it as "Need tests", specifically E2E tests. There's also the following note:
> 
> > The core functionality (expected markup) could be tested with integration tests. We should likely cover some of the known issues with e2e tests to cover nuanced visual regressions specific to each platform.
> 
> That's my rationale to retain it.

I see. I think this is similar to our discussion in https://github.com/wordpress-mobile/test-cases/pull/147#discussion_r1257256466. The reality is the known issues are captured in GitHub Issues and were not "covered" by the manual tests cases to begin with (the issue is a bug, it can't be covered until fixed). I think it is safe to remove that note from the document and will do so.

Reply:
Good point. I agree that if the only reason for keeping the test case is due to check known issues, we could remove it entirely. That said, my only concern would be about not covering visual testing on those blocks, but probably that's a common concern with any block, not these ones in specific.

@dcalhoun I'm going to merge this PR, but based on this convo, I think we could proceed with reviewing and approving the changes you worked on in the [unmerged branch](https://github.com/wordpress-mobile/test-cases/commit/8a5dbfc37b12fea72a57c051e88864b1b80a585c).

Reply:
> That said, my only concern would be about not covering visual testing on those blocks, but probably that's a common concern with any block, not these ones in specific.

Yes, this will be a judgement call that is more global. While I believe some blocks are important to test for visual regressions (Columns), I question the effort:value ratio for most blocks. 

> I'm going to merge this PR, but based on this convo, I think we could proceed with reviewing and approving the changes you worked on in the [unmerged branch](https://github.com/wordpress-mobile/test-cases/commit/8a5dbfc37b12fea72a57c051e88864b1b80a585c).

Sounds good. I'd like to expand the tests to cover splitting blocks from citation before remove it entirely. It's on my to-do list to follow up on this later.